gen-go/statsproto/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package statsproto contains generated Protobuf code for OpenCensus stats.
+package statsproto
```

Comment:
stats.proto hasn't been updated for the new design in https://github.com/census-instrumentation/opencensus-specs yet.

Reply:
Not sure what the new design is but let's fix that separately.

Reply:
Some of the data types have completely changed, such as MeasurementDescriptor, DistributionAggregation, and IntervalAggregation.  I agree that it should be updated separately from this PR, though.